### DISCUSSION TITLE
Mobile App

### DISCUSSION THREAD

**Discussion by astoilkov** [notaapp/talk#247] [Feature Requests]
We've started experimenting with a mobile app. You can upvote in this discussion or share your thoughts.

**Comment by borekb** [comment]
Would be great if it doesn't take away too much of your focus from the desktop app 😄.

Currently on mobile, I use iA Writer which can connect to GDrive and works similarly to Nota – I'm happy with it as far as UX goes but I also have to say that **I don't enjoy updating my notes on mobile**, the screen real estate is just too limited and I use it only in rare cases. If I have to create some notes when I'm on the go, I prefer to use **Google Keep** which is more mobile-friendly and I'll then transfer that note to my Markdown notes when I'm at a desktop computer.

So really, mobile app is not that important to me – I'd much rather have nota excellent on desktop and bad or non-existent on mobile than mediocre on both.

---

_Footnote about iA Writer:_

I now realize that maybe I'm just "lucky" and still use their [legacy & free Android app](https://play.google.com/store/apps/details?id=net.ia.iawriter) (ID `net.ia.iawriter`) while the [current one](https://play.google.com/store/apps/details?id=net.ia.iawriter.x) (ID `net.ia.iawriter.x`) is [paid](https://ia.net/pricing). I dislike their pricing model and wouldn't be happy at all paying $30 just to be able to access my notes on a phone. Nota's pricing is much more natural.

**Comment by davistech** [comment]
Moblle would be nice, but only after spending more time on adding missing features to the desktop app. Example, text folding (previously requested). I’m using Nota on a regular basis in conjunction with Obsidian and DEVONthink. All three access a common folder and provide complimentary functionality. I forgot the 4th leg of the stool — Marked 2, which provides additional export options. 

Keep up the good work!

**Reply by HusseinT2000** [reply]
I agree, i feel its premature for Nota to have a mobile app as the amount of functionality it would add wouldn't be worth the time spent.

**Reply by astoilkov** [reply]
@davistech I totally agree with you. Our primary focus is the desktop app.

BTW, today we shipped text folding (as an experiment). It's a slow roll update, so it may take a while to get to you.

**Comment by webwesen** [comment]
mobile is for accessing the notes on the go, i just use a git client. 
so... spending efforts on desktop-only takes my vote.

**Reply by erusev** [reply]
Thanks for your input! Do mind sharing which git client you use? Thanks!

**Reply by webwesen** [reply]
Working Copy

**Reply by erusev** [reply]
Thanks!

**Comment by minthemiddle** [comment]
I would only use read-only for notes and I have no good option for that now on Android. I'm not editing notes on the go.

**Comment by astoilkov** [comment]
Thanks for all the comments. That helps a lot.

Here is a little overview of our thinking. We believe in mainly writing your notes on the desktop. Nota is a **productivity** writing app. It does have shortcuts and refined workflows for doing things super quickly. This is just impossible to do on mobile.

However, we commonly want to access our notes on mobile. Not edit (maybe a little), but view them.

This is why we started experimenting with a mobile app where you can view your notes. For now, the most crucial aspect is quick access — the default view are notes (from your entire workspace) sorted by modified date, shortcuts for favorites, and a powerful search.

We may explore minimalistic editing at some point — button to edit in another app, block-level editing, or something we haven't thought of yet.

**Reply by minthemiddle** [reply]
100% my use case and wish what a Nota mobile app should focus on!

**Reply by borekb** [reply]
@astoilkov That sounds great!

I'd also like to add that for viewing only, Google Drive android app is not bad at all for that (if people use that; I have my notes inside `GDrive/Notes`).

**Reply by jeroenpardon** [reply]
Mobile isn’t just phones, I edit and write notes on my iPad all the time. And I most certainly take quick notes on my iPhone, which I often elaborate on when I’m on my desktop or on my iPad. I really love Nota and where it’s going, but the lack of a mobile app is definitely holding me back as far as switching to it as my main notes app.

**Comment by davistech** [comment]
> On Jul 8, 2022, at 6:42 AM, Antonio Stoilkov ***@***.***> wrote:
> 
> 
> @davistech <https://github.com/davistech> I totally agree with you. Our primary focus is the desktop app.
> 
> BTW, today we shipped text folding (as an experiment). It's a slow roll update, so it may take a while to get to you.
> 
> 
Great! I’m looking forward to this. Nota just keeps getting better!

**Comment by fedorovpishet** [comment]
Dream about mobile Nota.

I use iA Writer but hope about Nota on iPhone. With Daily Notes.

**Reply by fedorovpishet** [reply]
Ready to be a beta-tester)

**Reply by astoilkov** [reply]
Cool. We may have something in a month or two. I will let you know.

**Reply by minthemiddle** [reply]
Same here, very happy to become beta tester!
